web/src/components/data-entities/diver/__test__/DriverList.test.tsx

```diff
@@ -0,0 +1,63 @@
+// @ts-ignore
+import React from 'react';
+import {describe} from '@jest/globals';
+import {logRoles, render, screen, waitFor} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import DriverList from '../DiverList';
+import {BrowserRouter} from 'react-router-dom';
+import * as axiosInstance from '../../../../api/api';
+import {AxiosResponse} from 'axios';
+
+jest.setTimeout(10000);
+
+describe('<DiverList/>', () => {
+
+    let mockGetResult;
+
+    beforeAll(() => {
+        mockGetResult = jest.spyOn(axiosInstance, 'getResult');
+    });
+
+    afterEach(() => {
+        mockGetResult.mockRestore();
+    });
+
+    it('grid columns auto sizing after data load', async () => {
+        // Load sample data.
+        const canned = require('./diver.json');
```

Comment:
Please remove this file and use sample data. Don't use real data for tests since this is a public repo.

Reply:
Done